### ISSUE TITLE
When build criu in yocto ，need to build criu as normal user（not root）

### ISSUE THREAD

**Issue by lilongjiang1** [checkpoint-restore/criu#1997]
**Description**

**Steps to reproduce the issue:**
1.create criu source tar.gz

git clone --bare https://github.com/checkpoint-restore/criu.git --branch=criu-dev                        

 mv criu/ git2_github.com.checkpoint-restore.criu.git/                                                                       

tar czvf git2_github.com.checkpoint-restore.criu.git.tar.gz -C git2_github.com.checkpoint-restore.criu.git .      

touch git2_github.com.checkpoint-restore.criu.git.tar.gz.done 

cp    git2_github.com.checkpoint-restore.criu.git.tar.gz*  $yocto_root/poky/build/downloads/

2. change criu source version to  v3.17
modify SRCREV to 4f8f295e57e68740699479d12c1ad251e6dd859f  in $yocto_root/poky//meta-virtualization/recipes-containers/criu/criu_git.bb 

4. build criu  
login as normal user(not as root): tester
bitbake criu

**Describe the results you received:**
mkdir: cannot create directory '/var/lib/criu': Permission denied

**Describe the results you expected:**
no error info:Permission denied 
build criu successfully

**Additional information you deem important (e.g. issue happens only occasionally):**
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.6 LTS"
NAME="Ubuntu"
VERSION="18.04.6 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.6 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

**CRIU logs and information:**

<details><summary>CRIU full dump/restore logs:</summary>
<p>

```
(paste your output here)
```

</p>
</details>

<details><summary>Output of `criu --version`:</summary>
<p>

```
 criu --version
Version: 3.17
GitID: v3.17
```

</p>
</details>

<details><summary>Output of `criu check --all`:</summary>
<p>

```
(paste your output here)
```

</p>
</details>

**Additional environment details:**

**Comment by adrianreber** [comment]
What does bitbake do? We are not familiar with that tool so it is difficult for us to understand what the problem is.

**Comment by lilongjiang1** [comment]
bitbake used to build specified component @adrianreber

**Comment by adrianreber** [comment]
I still don't know which command bitbake runs to create that error message you are seeing. Can you provide s reproducer without bitbake?

**Comment by adrianreber** [comment]
I know that the RPM and DEB packages are also being built as non root without problems.

**Comment by lilongjiang1** [comment]
The key problem is criu build script create /var/lib/criu must as root，  Permission denied problem occured  when a normal user run criu build  script。

**Comment by lilongjiang1** [comment]
/var/lib/ is owned by root user ，only root user can write，normal user has no privilged to write to /var/lib/，any suggestion，please？@adrianreber

**Comment by adrianreber** [comment]
This is fixed in 3.17.1 by https://github.com/checkpoint-restore/criu/commit/a90a1d4827a07d68e5d4e73bb8122aa1ca38525a

**Comment by lilongjiang1** [comment]
thanks@adrianreber
